app/helpers/remote_execution_helper.rb

```diff
@@ -205,4 +205,11 @@ def documentation_button_rex(section = '')
       url,
       :rel => 'external', :class => 'btn btn-info', :target => '_blank')
   end
+
+  def template_input_header(template)
+    header = _('Template input')
+    header += ' ' + remove_child_link('x', f, {:rel => 'twipsy', 'data-title' => _('remove template input'), :'data-placement' => 'left',
```

Comment:
just wondering why mixing 'data-title' and :'data-placement' (string and symbol) types as keys